packages/fixie/src/voice.ts

```diff
@@ -112,11 +117,18 @@ export class VoiceSession {
     console.log('[voiceSession] warming up');
     this.audioStarted = false;
     this.started = false;
-    const url = this.params?.webrtcUrl ?? 'wss://wsapi.fixie.ai';
-    this.socket = new WebSocket(url);
-    this.socket.onopen = () => this.handleSocketOpen();
-    this.socket.onmessage = (event) => this.handleSocketMessage(event);
-    this.socket.onclose = (event) => this.handleSocketClose(event);
+    try {
+      const url = this.params?.webrtcUrl ?? 'wss://wsapi.fixie.ai';
+      this.socket = new WebSocket(url);
+      this.socket.onopen = () => this.handleSocketOpen();
+      this.socket.onmessage = (event) => this.handleSocketMessage(event);
+      this.socket.onclose = (event) => this.handleSocketClose(event);
+      this.changeState(VoiceSessionState.CONNECTING);
+    } catch (e) {
```

Comment:
can you `catch (e: Error) {`

Reply:
I tried this before, but eslint is strict on this for a good reason - there's no guarantee the thing you are catching here is, in fact, an Error object!

[
![Screenshot 2023-12-11 at 3 05 05 PM](https://github.com/fixie-ai/fixie-sdk-js/assets/996692/a1db8131-936b-4908-8cb9-48eb2908296e)
](url)

Reply:
yeah, sorry, forgot JS only lets you have a single catch()